### ISSUE TITLE
Setup automated Docker builds on Quay

### ISSUE THREAD

**Issue by jackzampolin** [blockstack/blockstack-core#576]
This is just waiting on this https://github.com/blockstack/blockstack-core/pull/572 to get merged into master

**Comment by jackzampolin** [comment]
This is now complete for browser. Waiting on pip builds in `/Dockerfile` to finish this for core.

**Comment by kantai** [comment]
The automated quay builds break due to broken submodule in our repository. Incoming PR to fix the submodule.

**Comment by jackzampolin** [comment]
Have the build running now. 

http://gph.is/1NZFbkg

**Comment by kantai** [comment]
Looks good to me
